vendor/vulkan/structs.odin

```diff
@@ -32,7 +36,20 @@ when ODIN_OS == .Windows {
 	}
 }
 
-CAMetalLayer :: struct {}
+// We want to use `vendor:x11/xlib` types so we need to match their build constraints.
+when ODIN_OS == .Linux || ODIN_OS == .FreeBSD || ODIN_OS == .OpenBSD {
```

Comment:
Instead of copying over targets and keeping them in sync. I would love to see a `xlib.IS_SUPPORTED` constant in xlib itself, similar to `time.IS_SUPPORTED` and `thread.IS_SUPPORTED`.

Reply:
Done